src/Nethermind/Ethereum.Trie.Test/TrieTests.cs

```diff
@@ -147,7 +148,7 @@ private void RunTest(TrieTest test, bool secure)
 
             TestContext.WriteLine(Surrounded(permutationDescription));
 
-            PatriciaTree patriciaTree = new PatriciaTree(_db, Keccak.EmptyTreeHash, false, true);
+            PatriciaTree patriciaTree = new PatriciaTree(_db, Keccak.EmptyTreeHash, false, true, NullLogManager.Instance);
```

Comment:
loggers inside the PatriciaTrie have been needed for a long time. Now used extensively in pruning diagnostics.